volatility/framework/symbols/windows/extensions/network.py

```diff
@@ -0,0 +1,297 @@
+# This file is Copyright 2019 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+#
+
+import enum
+import itertools
+import logging
+import socket
+
+from volatility.framework import objects, interfaces, exceptions
+from volatility.framework import exceptions
+from volatility.framework.objects import Array
+from volatility.framework.renderers import conversion
+from volatility.framework.symbols.wrappers import Flags
+from volatility.framework import renderers
+from typing import Dict, Tuple
+
+vollog = logging.getLogger(__name__)
+
+def inet_ntop(address_family: int, packed_ip: Array) -> str:
```

Comment:
So far we've just been using ObjectInterface rather than Array, but Array is more accurate, so I'd say leave it that way, and if it throws up too many errors in the type checker then we'll have to figure something out about it...

Reply:
What do you mean by type checker?

Reply:
So we try to use mypy against the codebase, to ensure all the type information makes sense (and throw up any bugs of weird types).  We haven't got the type model perfectly specific (mostly because most functions return `ObjectInterface` rather than specific types, even if we *assume* they'll always return an Array, or Struct, or whatever), which is why having such a specific type there might throw up issues, but as I say, we'll figure it out when that happens.  Happy for you to mark this as resolved if you want.  It's more my own thinking...  5:)

Reply:
Ok! I'll resolve this for now, but I'm open to updating the definition if need arises :)